### PR TITLE
Strong Parameters

### PR DESCRIPTION
**Pull request by phlipper**
- this implements `strong_parameters` support for `identities`

### CODE REVIEW

**Comment by phlipper** [comment]
fixes #24

File: app/controllers/thincloud/authentication/registrations_controller.rb [hunk 1]
```diff
@@ -81,12 +81,23 @@ def add_omniauth_identity_to_current_user
...
+
+    # Private: Provide strong_parameters support
+    # :token, :auth_key, :provider,
+    def identity_params
+      keys = [
+        :name, :email, :password,
+        :password_confirmation, :verification_token
+      ]
+
+      params.require(:identity).permit(*keys)
```
Comment (elskwid) [inline app/controllers/thincloud/authentication/registrations_controller.rb:100]: They just can't leave those hashes alone. Poor little data structure….

**Comment by elskwid** [comment]
:pineapple: :fire_engine: :shaved_ice:

File: app/controllers/thincloud/authentication/registrations_controller.rb (hunk 1 above)
Comment (supernullset) [inline app/controllers/thincloud/authentication/registrations_controller.rb:100]: Well, when you have a :hammer: ...

**Pull request merged**